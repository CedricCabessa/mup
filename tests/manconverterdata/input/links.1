.TH LINKS 1
.SH NAME
links - contains links
.SH DESCRIPTION

Link to
.BR foo (1)

Link to
.BR bar (3)
