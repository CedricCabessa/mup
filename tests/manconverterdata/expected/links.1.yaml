patterns:
    - <a href="/man/1/foo.1">foo\(1\)</a>
    - <a href="/man/3/bar.3">bar\(3\)</a>
    - <a href="/man/1/xbaz.1">xbaz\(1x\)</a>
